src/education.js

```diff
@@ -1,13 +1,29 @@
+//ul tag
+let eduTag = document.createElement("ul");
+eduTag.id = "edu-list";
+eduTag.className = "edu-list";
+
+//append ul tag to div
+let eduDiv = document.getElementById("edu-section");
+eduDiv.appendChild(eduTag).innerHTML;
+
+//li tag
+let eduList = document.createElement("li");
+eduList.id = "edu-items";
+eduTag.appendChild(eduList);
+
```

Comment:
This should all be scoped to the processData function. There's no reason for these to be global as they aren't being used elsewhere.
For all issues of scope, you need to figure out where the variables are being used and define them appropriately so all code that needs to use them can, but nothing more than that